interface/forms/soap/C_FormSOAP.class.php

```diff
@@ -65,10 +76,22 @@ function default_action_process()
         }
 
         if (empty($_POST['id'])) {
-            addForm($GLOBALS['encounter'], "SOAP", $this->form->id, "soap", $GLOBALS['pid'], $_SESSION['userauthorized']);
+            addForm(
+                $GLOBALS['encounter'],
+                "SOAP",
+                $this->form->id,
+                "soap",
+                $GLOBALS['pid'],
+                $_SESSION['userauthorized']
+            );
             $_POST['process'] = "";
         }
-
-        return;
+    }
+    /**
+     * @return string
+     */
+    public function getTemplatePath(): string
```

Comment:
change public to private

Reply:
@adunsulag in the Controller.class.php file you have this note // TODO: @adunsulag move these into src/
If I can shadow you on making this move. That would help me as a developer.

Reply:
@bradymiller I will begin testing today or tomorrow or soon. =^)8